services/dcs/src/dcs/adapters/outbound/daopub.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Adapter for publishing outbox events to other services."""
+
+from ghga_event_schemas.pydantic_ import NonStagedFileRequested
+from hexkit.protocols.daopub import DaoPublisher, DaoPublisherFactoryProtocol
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+from dcs.ports.outbound.daopub import OutboxPublisherFactoryPort
+
+
+class OutboxDaoConfig(BaseSettings):
+    """Configuration for the outbox DAO and publishing events"""
+
+    unstaged_download_event_topic: str = Field(
+        default=...,
+        description=(
+            "Name of the topic used for events indicating that a download was requested"
+            + " for a file that is not yet available in the outbox."
+        ),
+        examples=["file_downloads"],
+    )
+    unstaged_download_collection: str = Field(
+        default=...,
+        description=(
+            "The type used for event indicating that a download was requested"
+            + " for a file that is not yet available in the outbox."
+        ),
+        examples=["unstaged_download_requested"],
```

Comment:
Not sure if this has to be hyphenated instead

Reply:
It does, I've updated the description.